x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/scalar/string/RLike.java

```diff
@@ -5,17 +5,21 @@
  * 2.0.
  */
 
-package org.elasticsearch.xpack.esql.evaluator.predicate.operator.regex;
+package org.elasticsearch.xpack.esql.expression.function.scalar.string;
```

Comment:
++

I'm not completely convinced that the way our operators are scattered in different packages is correct (in particular `==`, `>=` etc. seem to be out of place in `.esql.evaluator`), so +1 to this change

Reply:
Yeah. I'm not sure it's all that important. I'm just trying to put functions together with their friends.